infogami/infobase/tests/pytest_wildcard.py

```diff
@@ -26,7 +28,8 @@ def test_wildcard():
     assert 1 == wildcard
     assert ["foo", 1, 2] == [wildcard, 1, 2]
 
-def pytest_funcarg__wildcard(request):
+@pytest.fixture
```

Comment:
Some of the test failures make me suspicious that this isn't working as intended, but need to dig in more